lib/strategy/consul/endpoint.ex

```diff
@@ -7,7 +7,7 @@ defmodule Cluster.Strategy.Consul.Endpoint do
 
   @callback build_url(URI.t(), Keyword.t()) :: URI.t()
 
-  @callback parse_response([map]) :: [ip]
+  @callback parse_response([map], list()) :: [ip]
```

Comment:
I think `Keyword.t()` fits better here.

```suggestion
  @callback parse_response([map], Keyword.t()) :: [ip]
```